package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "s47_nkr_plu2_0",
+  "version": "1.0.0",
+  "description": "PROJECT TITLE",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Consider adding more specific scripts for testing and building the project. This will make it easier for other developers to understand how to interact with your project. [important]